Website/index.js

```diff
@@ -0,0 +1,474 @@
+            });
+
+            nameChecker();
+        }
+    );
+}
+
+var blink = false;
+
+function recessTime(period) {
+    var icon = "<svg xmlns='http://www.w3.org/2000/svg' width='16' height='16' fill='currentColor' class='bi bi-record-circle mx-1 icon-blink' style='fill: #ff0000' viewBox='0 0 16 16'><path d='M8 15A7 7 0 1 1 8 1a7 7 0 0 1 0 14zm0 1A8 8 0 1 0 8 0a8 8 0 0 0 0 16z' /><path d='M11 8a3 3 0 1 1-6 0 3 3 0 0 1 6 0z' /></svg>";
+
+    function blinkText() {
+        blink = true;
+        $(".icon-blink").each(function () {
+            var elem = $(this);
+            setInterval(function () {
+                if (elem.css('visibility') == 'hidden') {
+                    elem.css('visibility', 'visible');
+                } else {
+                    elem.css("visibility", "hidden");
+                }
+            }, 700);
+        });
+    }
+
+    if (period === 1) {
+        $(icon).prependTo("#first_recess");
+    } else if (period == 2) {
+        $(icon).prependTo("#second_recess");
+        $("#first_recess").insertAfter("#fourth_recess");
+    } else if (period == 3) {
+        $(icon).prependTo("#third_recess");
+        $("#first_recess").insertAfter("#fourth_recess");
+        $("#second_recess").insertAfter("#first_recess");
+    } else if (period === 4) {
+        $(icon).prependTo("#fourth_recess");
+        $("#first_recess").insertAfter("#fourth_recess");
+        $("#second_recess").insertAfter("#first_recess");
+        $("#third_recess").insertAfter("#second_recess");
+    }
+
+    blinkText();
+}
+
+function update() {
+    $("#updated").text(jsonData["updated"]);
+
+    $("#weekend").hide();
+    var d = String($("#start").val());
+    var dd = new Date(d);
+    var first_date = new Date(Object.keys(jsonData["dates"])[0])
+    var lastDate = new Date(Object.keys(jsonData["dates"])[Object.keys(jsonData["dates"]).length - 1]);
+    if (dd.getDay() == 6 || dd.getDay() == 0 || jsonData["dates"][String($("#start").val())] == 11 || dd < first_date || dd > lastDate) {
+        $("#date").hide();
+        $.each(["#weekend", "#recess_duty_text", "#first_recess", "#second_recess", "#third_recess", "#fourth_recess", "#morning_duty_text", "#morning_duty_block"], function (i, v) {
+            $(v).hide();
+        });
+        updateDate();
+        $("#date").fadeIn('slow');
+        $("#weekend").delay(500).fadeIn('slow');
+        return;
+    }
+
+    $.each(["#date", "#recess_duty_text", "#first_recess", "#second_recess", "#third_recess", "#fourth_recess", "#morning_duty_text", "#morning_duty_block"], function (i, v) {
+        $(v).hide();
+    });
+
+    function today(date) {
+        var inputDate = new Date(date);
+        var todaysDate = new Date();
+        if (inputDate.setHours(0, 0, 0, 0) === todaysDate.setHours(0, 0, 0, 0)) {
+            return true
+        } else {
+            return false;
+        }
+    }
+
+    function updateBlockArrangement() {
+        var startTime = '06:00:00';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Davianlols6/SC-Duty-List/pullRequest?prid=8386818)
```suggestion
        var startTime = "06:00:00";
```